tests/job.yaml

```diff
@@ -1,30 +1,24 @@
----
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: namespace-cleaner-container-job
-  namespace: das
-  labels:
-    app: namespace-cleaner
+  name: namespace-cleaner-test-job
 spec:
   template:
-    metadata:
-      labels:
-        app: namespace-cleaner
     spec:
-      serviceAccountName: namespace-cleaner
+      serviceAccountName: default
```

Comment:
see comment about rbac

Reply:
I have set the correct serviceAccountName

```yaml
---
apiVersion: batch/v1
kind: Job
metadata:
  name: namespace-cleaner-test-job
  namespace: das
spec:
  template:
    spec:
      serviceAccountName: namespace-cleaner
```